tango_simlib/utilities/fandango_json_parser.py

```diff
@@ -114,11 +115,13 @@ def preprocess_command_types(self, command_data):
                         {CMD_PROP_MAP[cmd_prop]: cmd_prop_value}
                     )
                 except KeyError:
-                    MODULE_LOGGER.info(
-                        "The property '%s' cannot be translated to a corresponding "
-                        "parameter in the TANGO library",
-                        cmd_prop,
-                    )
+                    invalid_command_properties.add(cmd_prop)
+
+        MODULE_LOGGER.debug(
+            "The properties '%s' cannot be translated to a corresponding "
+            "parameters in the TANGO library",
+            invalid_command_properties,
+        )
```

Comment:
This is much better. That's where most of the noisy logs were coming from.

Should we not wrap this in an `if block` like on line 158?

Reply:
Thanks. I will do that.